presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -112,7 +116,7 @@ public void shouldExecuteQueryWithSelectedCatalogAndSchema()
     public void shouldSetTimezone()
             throws SQLException
     {
-        if (connection instanceof PrestoConnection) {
+        if (usingFacebookJdbcDriver()) {
             String timeZoneId = "Indian/Kerguelen";
```

Comment:
Simba sets the timezone via a JDBC URL parameter, so there really isn't any equivalent.